getnacelle/nuxt-reference-store
# Ld 1184  newsletter

**What This PR Does**
- Builds out newsletter component
- Wires up newsletter to pull data from Sanity

**How To Test**
- Spin up branch locally and see that newsletter looks good.
- Spin up Sanity studio locally on branch (LL-1183_studio-scaffold) and see that the content matches that on page
- If you want you can make a change, re-index and see that it show up (I've noticed delays here but eventually it looks good)

**Troubleshooting**
- Make sure you are using the new space credentials
